lib/internal/util.js

```diff
@@ -53,6 +53,16 @@ const experimentalWarnings = new SafeSet();
 
 const colorRegExp = /\u001b\[\d\d?m/g; // eslint-disable-line no-control-regex
 
+
+let uvBinding;
+
+function lazyUv() {
+  if (!uvBinding) {
+    uvBinding = internalBinding('uv');
+  }
```

Comment:
```suggestion
  uvBinding ??= internalBinding('uv');
```

Reply:
I changed it, even though I think assignment and returns shouldn't be mixed.